lms/djangoapps/courseware/module_render.py

```diff
@@ -679,6 +713,8 @@ def rebind_noauth_module_to_user(module, real_user):
         reverse('jump_to_id', kwargs={'course_id': text_type(course_id), 'module_id': ''}),
     ))
 
+    block_wrappers.append(partial(display_access_messages, user))
```

Comment:
is this going to surface error messages to the user?  
should this be wrapped in a flag?  
in a similar situation product decided not to surface the other error messages that were showing up on the dashboard

Reply:
I pulled this out of the code.  The displaying error messages to user is going to part of another task.